components/Meta/index.js

```diff
@@ -1,17 +1,21 @@
 import Head from 'next/head'
 import metaShape from './shape'
+import metaDefaults from 'constants/metadata'
```

Comment:
Yeah this does strike me as kind of weird. So you're allowing defaults to be passed into the `Meta` component but you're allowing for a default level beyond that in case that's not passed?

Reply:
@bretmorris yeah it could be a little redundant, but this is in case there are no values set in prismic for default metadata. Like maybe someone doesn't set it on the project, or the client deletes a field value by accident? Just like a final backstop